### PR TITLE
Documentation and bug/format fixes

### PR DESCRIPTION
**Pull request by JeremiasLiew**
Added the PPP
Fix exception handling for getting card index and adding event
Fix typos and formatting issues
Remove option to add dates in a yyyy MM dd format as it is not working properly

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -162,11 +162,11 @@ Creates a new event and adds it to the app.
...
-* The `DATE` formats that are accepted are:
-    1. dd-MM-yy, dd-MM-yyyy, yyyy-MM-dd 
-    2. dd/MM/yy, dd/MM/yyyy, yyyy/MM/dd
-    3. dd.MM.yy, dd.MM.yyyy, yyyy.MM.dd
-    4. dd-MMM-yy, dd-MMM-yyyy
+* The accepted `DATE` formats are:
+    1. dd-MM-yy, dd-MM-yyyy
+    2. dd/MM/yy, dd/MM/yyyy
+    3. dd.MM.yy, dd.MM.yyyy
+    4. dd-MMM-yy, dd-MMM-yyyy (e.g. 1-May-2020)
```
Comment (kcubey) [inline docs/UserGuide.md:169]: You may want to indicate where it is number and where it is text? Just a suggestion haha
e.g. M indicates number and m indicates word format.

**Review by kcubey — APPROVED** [review]
Looks fine! Just take note of the suggestion 👍

**Review by alexlim510 — APPROVED** [review]
LGTM

**Comment by kcubey** [comment]
References #122

**Review by JeremyTakigawa — APPROVED** [review]

**Pull request merged**